src/fastoad/notebooks/02_explanation/modules/pure_python/mass/sub_components/compute_wing_mass.py

```diff
@@ -0,0 +1,31 @@
+def compute_wing_mass(mtow, aspect_ratio, wing_area):
+    """
+    Computes the wing mass based on the MTOW, its area and aspect ratio
+
+    :param wing_area: Wing area, in m2
+    :param aspect_ratio: Wing aspect ratio, no unit
+    :param mtow: Max Take-Off Weight, in kg
+
+    :return wing mass: the wing_mass, in kg
+    """
+
+    # Let's start by converting the quantities in imperial units
+    mtow_lbs = mtow * 2.20462
+    wing_area_ft2 = wing_area * 10.7639
```

Comment:
I know usage of scipy is for next notebook, but let's open the world of some readers :)

with
```python
import scipy.constants as sc
```
```suggestion
    mtow_lbs = mtow / sc.lb
    wing_area_ft2 = wing_area / sc.foot**2
```